core/source/org/libreoffice/ide/eclipse/core/wizards/NewUnoProjectWizard.java

```diff
@@ -237,9 +237,6 @@ public IWizardPage getNextPage(IWizardPage pPage) {
                     } else {
                         // Could be null
                         next = mServiceSet.getPage(ServiceWizardSet.SERVICE_PAGE_ID);
-                        if (mMainPage.getChosenLanguage().toString().contains("python")) {
-                            next = null;
-                        }
```

Comment:
The removal of this if condition makes sure that the Service and the interface page show up in the New Extension Project Wizard